CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.10.0 17th May 2024
```

Comment:
Not correct right?

Reply:
Yeah, [this](https://github.com/onfido/onfido-openapi-spec/pull/80) should fix that ;)

Reply:
PR updated ;)